tests/core/config/models/bdk_authentication_config_test.py

```diff
@@ -45,7 +45,7 @@ def test_certificate_configuration():
     assert authentication_config.is_certificate_configuration_valid() is False
 
     # No password with certificate
-    authentication_config.certificate.path = None
+    authentication_config.certificate.path =None
```

Comment:
supper picky, sorry: please add a space after the equal sign. 
you can (shift + option + command + L) on pycharm to reformat the file and you will never have these again 😄